### PR TITLE
Jon J sql crud diagnostic

### PR DESCRIPTION
**Pull request by modularjon**

### CODE REVIEW

**Comment by modularjon** [comment]
already submitted, but id should be

id SERIAL

not

id integer PRIMARY KEY

on line 9

File: diagnostic.sql [hunk 1]
```diff
@@ -5,24 +5,37 @@ CREATE DATABASE "sql-crud-assessment";
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname, given_name, gender, height, weight, born_on, eye_color, hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
--- your code here
+CREATE TABLE licensees (
+    id integer PRIMARY KEY
```
Comment (RealWeeks) [inline diagnostic.sql:9]: serial, but I see you caught you mistake already

File: diagnostic.sql [hunk 2]
```diff
@@ -5,24 +5,37 @@ CREATE DATABASE "sql-crud-assessment";
...
 
 -- After completing questions 1 and 2 uncomment the following to load data from `data/licensees.csv` into the `licensees` table.
 
--- \copy licensees(surname,given_name,gender,height,weight,born_on,eye_color,hair_color) FROM './data/licensees.csv' WITH (FORMAT csv, HEADER true)
+\copy licensees(surname,given_name,gender,height,weight,born_on,eye_color,hair_color) FROM './data/licensees.csv' WITH (FORMAT csv, HEADER true)
 
 -- Question 3. Write a query to get all attributes of licensees with Hazel eye color and Brown or Black hair color.
 
--- your code here
+SELECT * FROM licensees WHERE eye_color = 'Hazel' AND hair_color = 'Brown' OR 'Black'
```
Comment (RealWeeks) [inline diagnostic.sql:31]: need to specify hair_color again here

**Comment by RealWeeks** [comment]
4/5.  Great job, keep up the strong effort.

**Pull request closed without merging**